.rubocop.yml

```diff
@@ -285,6 +318,11 @@ Style/AccessorGrouping:
   Description: 'Checks for grouping of accessors in `class` and `module` bodies.'
   Enabled: true
 
+Style/ArgumentsForwarding:
+  Description: 'Use arguments forwarding.'
+  StyleGuide: '#arguments-forwarding'
+  Enabled: false
+
```

Comment:
Ruby 2.7 `...` new forwarding syntax which I don't fully understand yet and breaks my editor syntax's checker :D Disabled in the meantime

Reply:
> which I don't fully understand yet

The best use case would be our service objects:

```ruby
# Before
def self.call(arg1, arg2, kwarg1:)
  new(arg1, arg2, kwarg1: kwargs1).call
end

# After
def self.call(...)
  new(...).call
end
```

Though I'm actually not a fan of it for this specific use-case, the whole reason I want duplication is to make the signature of the main interface more explicit.

Reply:
Exactly, I think the old syntax is clearer. "..." can mean anything